lib/sys_info.dart

```diff
@@ -31,16 +35,19 @@ class SystemInfo {
     final osVersion = await _getOsVersion(deviceInfo);
 
     return SystemInfo._(
-        osName: osName,
-        osVersion: osVersion,
-        locale: Platform.localeName,
-        buildNumber: packageInfo.buildNumber,
-        appVersion: packageInfo.version);
+      osName: osName,
+      osVersion: osVersion,
+      locale: Platform.localeName,
+      buildNumber: packageInfo.buildNumber,
+      appVersion: packageInfo.version,
+    );
   }
 
   /// Returns the name of the operating system.
   static Future<String?> _getOsName(DeviceInfoPlugin deviceInfo) async {
-    if (Platform.isAndroid) {
+    if (kIsWeb) {
+      return "Web";
```

Comment:
Yeah I can set `osName` and `osVersion` to empty if we are running on web. 

I am not quite sure about the last part regarding the user-agent on the http request, though. 
Do you mean we should also skip this line when we are on web? https://github.com/aptabase/aptabase_flutter/blob/6e88646ff645be995427ab5d6375624d33fcf123/lib/aptabase_flutter.dart#L83

Reply:
Yes, we can skip that line. I'm assuming that universal_io will use the user agent from the browser if we don't override. Because if it does, then our server will parse the correct user agent.

Reply:
Done